scripts/lint.sh

```diff
@@ -51,6 +51,12 @@ function test_license_header {
   "${files[@]}"
 }
 
+function test_single_import {
+  if grep -R -zo -e 'import (\n\t".*"\n)' .; then
+    return 1
```

Comment:
Should also add a log to say why the linter failed.

Reply:
It does print it:
```
START: 'single_import' at Thu May  4 17:37:48 EDT 2023
./snow/consensus/snowball/binary_slush.go:import (
        "fmt"
)
./snow/consensus/snowball/binary_snowflake.go:import (
        "fmt"
)
./snow/consensus/snowball/binary_snowball.go:import (
        "fmt"
)
FAIL: 'single_import' failed at Thu May  4 17:37:50 EDT 2023
```